searx/webapp.py

```diff
@@ -1094,6 +1120,11 @@ def config():
     })
 
 
+@app.route('/config/http')
```

Comment:
Why expose this endpoint? It does not seem useful to anyone besides developers.

Reply:
Removed.